src/main/java/lawrence/task/Task.java

```diff
@@ -0,0 +1,77 @@
+package lawrence.task;
+
+/**
+ * Represents a real-life task that needs to be completed.
+ */
+public abstract class Task {
+    private boolean complete;
```

Comment:
boolean variables should sound like booleans. e.g. isComplete

Reply:
Will rename to `isComplete` since there is no need for a getter method `isComplete()` yet.
I had created a forum post to clarify on this, but it seems highly contentious, and implementation seems to be down to personal perference (as long as it is consistent).

See: https://github.com/nus-cs2103-AY2425S1/forum/issues/210